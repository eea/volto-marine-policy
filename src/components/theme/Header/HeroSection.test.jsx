import React from 'react';
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';
import { Provider } from 'react-intl-redux';

import HeroSection from './HeroSection';

const mockStore = configureStore();

test('renders Hero section component', () => {
  const store = mockStore({
    intl: {
      locale: 'en',
      messages: {},
    },
  });

  const component = renderer.create(
    <Provider store={store}>
      <HeroSection
        image_caption="caption"
        image_url="https://www.google.com"
        content_description="description"
        content_title="title"
      />
    </Provider>,
  );
  const json = component.toJSON();
  expect(json).toMatchSnapshot();
});
