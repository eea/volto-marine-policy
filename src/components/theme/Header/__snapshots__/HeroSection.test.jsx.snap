// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Hero section component 1`] = `
<div
  className="herosection"
>
  <div
    className="herosection-content-wrapper"
  >
    <div
      className="herosection-content-image document-image"
      style={
        Object {
          "backgroundImage": "url(https://www.google.com)",
        }
      }
    />
    <div
      className="herosection-content-image-overlay"
    />
    <div
      className="ui container herosection-content"
    >
      <h1
        className="herosection-content-title"
      >
        title
      </h1>
      <p
        className="herosection-content-description"
      >
        description
      </p>
      <p
        className="herosection-content-image-caption"
      >
        caption
      </p>
    </div>
  </div>
</div>
`;
